src/luascript.cpp

```diff
@@ -504,14 +504,14 @@ int LuaScriptInterface::luaErrorHandler(lua_State* L)
 	return 1;
 }
 
-bool LuaScriptInterface::callFunction(int params)
+bool LuaScriptInterface::callFunction(int params, bool defaultReturn /* = false */)
```

Comment:
It's because in this one, it'll be the default return value of the function being called, in `LuaScriptInterface::getBoolean` it's the value of the variable being read.
I think it's more explicit this way what that default represents